worlds/ufo50/__init__.py

```diff
@@ -16,14 +16,24 @@
 from .games.porgy import items, locations, regions
 from .games.vainger import items, locations, regions
 from .games.night_manor import items, locations, regions
+from worlds.LauncherComponents import components, Component, launch_subprocess, Type as ComponentType
+
+
+def launch_client(*args: str):
+    from .Client import launch
```

Comment:
```suggestion
    from .client import launch
```
To match the rest of the files, this should probably use a lowercase c, but also I'm not gonna die on this hill lol